test/format_strftime_test.exs

```diff
@@ -17,6 +17,20 @@ defmodule DateFormatTest.FormatStrftime do
     end)
   end
 
+  test "success!" do
+    date = Timex.to_datetime({2013,03,02})
+
+    formatter = Timex.Format.DateTime.Formatters.Strftime
+    assert "2013-03-02" == formatter.format!(date, "%Y-%m-%d")
+  end
+
+  test "error tuple" do
+    date = Timex.to_datetime({2013,03,02})
+
+    formatter = Timex.Format.DateTime.Formatters.Strftime
```

Comment:
Same as the above

Reply:
Let's give this test a more descriptive name, like "error returned when invalid directive is given"